src/status/helpers.js

```diff
@@ -0,0 +1,22 @@
+  const moment = require('moment');
+  
+  function occuredWithinCurrentWeek(date) {
+      /*TODO: add timezone*/
+    const now = moment();
```

Comment:
you want to make helper functions as stateless as possible and move the state (`now`) to where this function is called